themes/default/content/docs/pulumi-cloud/deployments/reference.md

```diff
@@ -226,3 +226,11 @@ By default, the deployment executor will attempt to install dependencies for you
 This is enabled by skipping the default dependency installation step (under Advanced Settings in the UI), and setting a few pre-run commands and environment variables.
 
 ![Pulumi UI - Node Version](../ui-node-version.png)
+
+### Building Docker images for another architecture
+
+By default, the deployment executor will only build Docker images for the x86_64 architecture, if you need to build images for another architecture (e.g. linux/arm64), you can add the architecture as a pre-run command such as:
```

Comment:
This isn't quite right, to run _steps_ in a Dockerfile for another architecture requires an emulator to be installed. There are many tools (Rust, Go, C with the appropriate tools installed, etc.) that support cross-compilation and can build images for `linux/arm64` without any emulator by using `FROM --platform=$BUILDPLATFORM ...` and creating a statically linked binary that runs on the target platform.

Suggest two changes:
1. Saying that the deployment engine uses x86-64 (aka `linux/amd64`) hosts.
2. Suggest that if users are building binaries for another platform, whether via the `Command` provider, `Docker`, they reference the documentation for building cross platform tools.
3. Suggest the command below to install emulators for Docker, linking to Docker documentation.

Reply:
@sicarul were you able to update this section of the PR? I see this has been re-opened.

Reply:
@EvanBoyle told me to send the PR through Slack, i sent the updated changes merging the latest version